src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DocumentMapping/DefaultLSPDocumentMappingProvider.cs

```diff
@@ -79,7 +64,7 @@ internal class DefaultLSPDocumentMappingProvider : LSPDocumentMappingProvider
             Kind = languageKind,
             RazorDocumentUri = razorDocumentUri,
             ProjectedRanges = projectedRanges,
-            MappingBehavior = mappingBehavior,
+            MappingBehavior = LanguageServerMappingBehavior.Strict,
```

Comment:
Okay, this method was not technically unused, but it was only ever called by the method directly above this, and that passed in Strict, so here we are 😁